test/shmemx/shmemx_team_collect.c

```diff
@@ -0,0 +1,47 @@
+/*
+ *  This test program is derived from an example program in the
+ *  OpenSHMEM specification.
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <shmem.h>
+#include <shmemx.h>
+
+int main(void)
+{
+   static long lock = 0;
+
+   shmem_init();
+   int me = shmem_my_pe();
+   int npes = shmem_n_pes();
+   int my_nelem = me + 1; /* linearly increasing number of elements with PE */
+   int total_nelem = (npes * (npes + 1)) / 2;
+
+   int* source = (int*) shmem_malloc(npes*sizeof(int)); /* symmetric alloc */
+   int* dest = (int*) shmem_malloc(total_nelem*sizeof(int));
+
+   for (int i = 0; i < my_nelem; i++)
+      source[i] = (me * (me + 1)) / 2 + i;
+   for (int i = 0; i < total_nelem; i++)
+      dest[i] = -9999;
```

Comment:
Should we keep `-1` or `INT_MIN` instead? I understand, it does not make any difference, but `-9999` seems a specific value.

Reply:
Since it's an upstream test, I might suggest we keep it to avoid future confusion and having to include  `limits.h`.  But I'm fine with it if you insist ;)